configure.ac

```diff
@@ -1,10 +1,13 @@
 AC_PREREQ([2.69])
 AC_INIT([iBASIC], [0.1], [ian@mnementh.co.uk], [ibasic], [https://github.com/spyro2/ibasic])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign subdir-objects])
```

Comment:
Note that this `-Wall -Werror` applies to `automake`, not to the C compiler. The `foreign` is to say that we don't really care about all the GNU-specific extra files and oddities (it's "foreign to the GNU project", as in not part of it). The `subdir-objects` option is going to be the new default at some point, where you have just one `Makefile.am` file at the top, and it goes 10x faster. 😃